CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ set( CXX_STANDARD_REQUIRED ON)
 
 set(VERSION_MAJOR 1)
 set(VERSION_MINOR 8)
-set(VERSION_PATCH 0)
+set(VERSION_PATCH 1)
```

Comment:
This is not relevant to our pipeline work, so let's make this a separate PR to `release/1.8.x`.